babel.config.js

```diff
@@ -10,6 +10,7 @@ if (process.env.BABEL_ENV === 'es') {
     [
       '@babel/preset-env',
       {
+        bugfixes: true,
```

Comment:
Doesn't seem to do anything for us at the moment. Leaving this in to have a smoother babel 8  upgrade path.